services/worker/src/worker/job_runners/dataset/is_valid.py

```diff
@@ -48,8 +54,24 @@ def get_job_type() -> str:
     def get_job_runner_version() -> int:
         return PROCESSING_STEP_DATASET_IS_VALID_VERSION
 
+    def __init__(
+        self,
+        job_info: JobInfo,
+        app_config: AppConfig,
+        processing_step: ProcessingStep,
+        processing_graph: ProcessingGraph,
+    ) -> None:
+        super().__init__(
+            job_info=job_info,
+            app_config=app_config,
+            processing_step=processing_step,
+        )
+        self.processing_graph = processing_graph
+
     def compute(self) -> JobResult:
```

Comment:
Maybe you can use `CompleteJobResult` and change `compute_is_valid_response` return type to `DatasetIsValidResponse`.
So then, you return here CompleteJobResult(compute_is_valid_response....)
You don't need 1.0 anymore if it is a CompleteJobResult.

Reply:
Right! I'll change it

Reply:
Done, thanks for the idea!